### DISCUSSION TITLE
Documentation appears to be out of date

### DISCUSSION THREAD

**Discussion by FeldrinH** [orientechnologies/orientdb#9946] [Q&A] [answered]
The documentation that is easily discoverable and linked everywhere is http://orientdb.org/docs/, which appears to be noticeably out of date. The linked documentation has OrientDB 3.0 as stable and 3.1 beta as latest. Looking at GitHub the latest stable version appears to be 3.2.17.

Why is the public documentation so out of date? Is there a more up to date version of the documentation available?

**Comment by tglman** [comment, marked as answer]
Hi,

yep I think the default link is still pointing to some old docs, if you open: https://orientdb.com/docs/3.2.x/ you should see the last version.

bye

**Reply by FeldrinH** [reply]
Interesting that on http://www.orientdb.com/docs/last/index.html the little version dropdown on the top left doesn't even show the 3.2.x version as an option.

**Reply by FeldrinH** [reply]
![image](https://user-images.githubusercontent.com/5256211/225907897-98ae96eb-e82d-4663-a0a9-bceed93a5ad7.png)
